### PR TITLE
View subject in course_subject

### PR DESCRIPTION
**Pull request by dhung133**

### CODE REVIEW

File: app/views/supervisor/course_subjects/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<div class="col-xs-6 col-md-6 text-center">
+  <h1><%= @course.name %></h1>
+  <h2><%= t "subjects.show" %>: <%= @subject.name %></h2>
+  <h3><%= @subject.description %></h3>
+</div>
+<div class="col-xs-12 col-md-6">
+<h3><%= t "subjects.tasks" %> (<%= @subject.tasks.count %>)</h3>
+  <% if @subject.tasks.any? %>
+    <ol>
+      <% @subject.tasks.all.each do |task| %>
```
Comment (NgengChhengkim) [inline app/views/supervisor/course_subjects/show.html.erb]: `@subject.tasks.each`

File: app/views/supervisor/course_subjects/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<%= provide :title, @subject.name %>
+<div class="col-xs-6 col-md-6 text-center">
+  <h1><%= @course.name %></h1>
+  <h2><%= t "subjects.show" %>: <%= @subject.name %></h2>
+  <h3><%= @subject.description %></h3>
+</div>
+<div class="col-xs-12 col-md-6">
+<h3><%= t "subjects.tasks" %> (<%= @subject.tasks.count %>)</h3>
+  <% if @subject.tasks.any? %>
```
Comment (NgengChhengkim) [inline app/views/supervisor/course_subjects/show.html.erb]: Khong can kiem tra co chay dung ko?

File: app/views/supervisor/course_subjects/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<%= provide :title, @subject.name %>
+<div class="col-xs-6 col-md-6 text-center">
+  <h1><%= @course.name %></h1>
+  <h2><%= t "subjects.show" %>: <%= @subject.name %></h2>
+  <h3><%= @subject.description %></h3>
+</div>
+<div class="col-xs-12 col-md-6">
+<h3><%= t "subjects.tasks" %> (<%= @subject.tasks.count %>)</h3>
```
Comment (NgengChhengkim) [inline app/views/supervisor/course_subjects/show.html.erb]: bo ()

Comment (hellovietnam93) [inline app/views/supervisor/course_subjects/show.html.erb]: ident

**Pull request merged**